subjects/devops/better-cat/README.md

```diff
@@ -0,0 +1,64 @@
+## better-cat
+
+### Instructions
+
+Create a script `better-cat.sh` that will behave similarly to the cat command, but with additional functionality.
+
+The script should accept any number of filenames as arguments, and it should print the content of each file to the standard output. If no arguments are provided, the script should print the content of all files in the current directory.
+
+The script should also support the following options:
+
+- -c: If this option is provided, the script should exclude all lines that begin with the # character.
+- -l: If this option is provided, the script should print the line number before each line of output.
+- -r: If this option is provided, the script should print a summary line at the end of the output, indicating the total number of lines and characters printed.
+
+If both -c and -l are provided, the line number should include only non-comment lines.
+
+### Usage
+
+```console
+$ bash better-cat.sh file1.txt
+Here would be the content of file1
+
+$ bash better-cat.sh file1.txt file2.txt
+Here would be the content of file1
+Here would be the content of file2
+
+$ bash better-cat.sh
+Here would be the content of file1
+Here would be the content of file2
+Here would be the content of file3
+
+$ bash better-cat.sh -c commented.txt
```

Comment:
I find the below example a bit confusing: you are still referring to `file3` but have passed the argument `commented.txt`.
Additionally, which are the commented lines that should not appear - it might be enough to only skip some of the lines of the 10 that are here

Reply:
Fixed. Thanks.